AlpsSDK/ModelExtensions.swift

```diff
@@ -103,3 +104,14 @@ extension ProximityEvent {
         self.distance = distance
     }
 }
+
+extension CLProximity {
+    
+    static let allValues = [unknown, immediate, near, far]
```

Comment:
Is there any reason why it's static variable?

Reply:
Tried without, Xcode can't compile.
Saw this on stackoverflow. I was using this to loop over CLProximity enum, but with refactoring we don't need this anymore.